src/python/evaluation/xlsx_run_tool.py

```diff
@@ -0,0 +1,151 @@
+import argparse
+import logging.config
+import os
+import re
+import sys
+import traceback
+from pathlib import Path
+from typing import NoReturn
+
+sys.path.append('')
+sys.path.append('../../..')
+
+import pandas as pd
+from openpyxl import Workbook
+from src.python import MAIN_FOLDER
+from src.python.evaluation import ScriptStructureRule
+from src.python.evaluation.evaluation_config import ApplicationConfig
+from src.python.evaluation.support_functions import remove_sheet
+from src.python.review.common.subprocess_runner import run_in_subprocess
+from src.python.review.reviewers.perform_review import OutputFormat
+
+logger = logging.getLogger(__name__)
+
+
+def configure_arguments(parser: argparse.ArgumentParser) -> NoReturn:
+    parser.add_argument('data_path',
+                        type=lambda value: Path(value).absolute(),
+                        help='Local XLSX-file path. '
+                             'Your XLSX-file must include column-names: \"code\" and \"lang. '
+                             'Acceptable values for \"lang\" column are: python3, java8, java11, kotlin.')
+
+    parser.add_argument('-t', '--tool_path',
+                        default=Path('src/python/review/run_tool.py').absolute(),
+                        type=lambda value: Path(value).absolute(),
+                        help='Path to script to run on files.')
+
+    parser.add_argument('--traceback', '--traceback',
+                        help='If True – grades are substituted with the full inspector feedback.',
+                        default=False,
+                        type=bool)
+
+    parser.add_argument('--folder_path', '--folder_path',
+                        help='An absolute path to the folder where file with evaluation results'
+                             'will be stored.'
+                             'Default is \"hyperstyle/src/python/evaluation/results\"',
+                        default=None,
+                        type=str)
+
+    parser.add_argument('--file_name', '--file_name',
+                        help='Filename for that will be created to store inspection results.'
+                             'Default is \"results.xlsx\"',
+                        default='results.xlsx',
+                        type=str)
+
+    parser.add_argument('-f', '--format',
+                        default=OutputFormat.JSON.value,
+                        choices=OutputFormat.values(),
+                        type=str,
+                        help='The output format of inspectors traceback. '
+                             'Default is JSON.'
+                             'More details on output format options in README.md')
+
+
+def create_dataframe(config) -> pd.DataFrame:
+    report = pd.DataFrame(
+        {
+            "language": [],
+            "code": [],
+            "grade": [],
+        },
```

Comment:
Using strings in multiple places instead of variables is a potential error location. Also, why are you adding the new dataframe below to the dateframe at all? why don't you just add a row?